src/common/preferences/preferences.html

```diff
@@ -35,53 +35,51 @@
 				<div class="pane-image"><img src="../images/prefs-general.png" alt=""/></div>
 				<div class="pane-label">General</div>
 			</div>
+			<div id="pane-proxies" class="pane" style="display: none">
+				<div class="pane-image"><img src="../images/prefs-proxies.png" alt=""/></div>
+				<div class="pane-label">Proxies</div>
+			</div>
 			<div id="pane-advanced" class="pane">
 				<div class="pane-image"><img src="../images/prefs-advanced.png" alt=""/></div>
 				<div class="pane-label">Advanced</div>
 			</div>
 		</div>
 		<div id="content-general" class="content">
 			<div class="group">
-				<div class="group-title">Zotero Standalone</div>
+				<div class="group-title">Zotero Status</div>
 				<div class="group-content">
-					<p>Zotero Standalone is currently <span id="general-span-standalone-status"></span>.</p>
+					<p>Zotero is currently <span id="general-span-standalone-status"></span>.</p>
 					<p><input type="button" value="Update Status" id="general-button-update-standalone-status"/></p>
 				</div>
 			</div>
 			<div class="group">
-				<div class="group-title">Save via Server</div>
+				<div class="group-title">Save to Zotero.org</div>
```

Comment:
![image](https://cloud.githubusercontent.com/assets/5899315/21421374/2cc91266-c83b-11e6-9b51-6fde3e877a84.png)
![image](https://cloud.githubusercontent.com/assets/5899315/21421384/371e6a9a-c83b-11e6-83f3-25e230b0b1e7.png)
This is a section title, looks better capitalized imo.